obsreport/obsreport_processor.go

```diff
@@ -78,7 +79,8 @@ func (por *Processor) TracesAccepted(ctx context.Context, numSpans int) {
 // TracesRefused reports that the trace data was refused.
 func (por *Processor) TracesRefused(ctx context.Context, numSpans int) {
```

Comment:
Same question, should this return error? Does otel return error?

Reply:
I don't think otel does as of today (guessing [this](https://pkg.go.dev/go.opentelemetry.io/otel/metric#Int64UpDownCounter.Add) is equivalent for example?). For the obsreport errors I was just following what we do for other components e.g. see https://github.com/open-telemetry/opentelemetry-collector/blob/89e230ef02b469f7f45f9bcdbbfa7ce0f7c0c717/obsreport/obsreport_exporter.go#L108-L114

Reply:
I saw that, but curios if we should return error because that will be the case when switching to otel.

Reply:
Seems not to be the case so we are good.